apps/dashboard/src/components/workflow-editor/schema.ts

```diff
@@ -35,7 +35,12 @@ export const workflowMinimalSchema = z.object({
   description: z.string().max(MAX_DESCRIPTION_LENGTH).optional(),
 });
 
-export const workflowSchema = workflowMinimalSchema.extend({
+export const configureStepSchema = z.object({
+  name: z.string().min(1).max(MAX_NAME_LENGTH),
+  stepId: z.string(),
+});
+
+export const workflowSchema = configureWorkflowSchema.extend({
```

Comment:
I suggest we define a single workflowSchema with all the fields and then we [pick/omit per use case](https://github.com/colinhacks/zod?tab=readme-ov-file#pickomit). That way, we will only have `stepSchema` and `workflowSchema` in Zod.

Reply:
actually, i don't think we're going to need any more, since the forms are only the simple cases now.